app/views/items/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for [user, user.items.new], remote: true do |f| %>
```

Comment:
[Fixed in this commit.](https://github.com/sanjayypatel/blocitoff/commit/3d226a22d890117d18d114305f06999f45529060)